waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -246,6 +246,18 @@
                  :waiter/pod-expired pod-expired})
       true)))
 
+(defn generate-port-with-offset
+  [service-id pod-base-port direction has-sidecar]
```

Comment:
Please pass the offset in here.

We should have a constant defined for the offset.